bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/dto/ScheduleCommand.java

```diff
@@ -55,7 +58,7 @@ public String getMethod() {
      *
      * @return request body
      */
-    public String getBody() {
-        return body.toString();
+    public @Nullable String getBody() {
+        return body != null ? body.toString() : "";
```

Comment:
Apparently, your return cannot be null

Reply:
Luckily this class is not needed at all. I removed all references.